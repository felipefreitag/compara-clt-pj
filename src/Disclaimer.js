import React from 'react';
import { Card, CardHeader, CardText } from 'material-ui/Card'

const Disclaimer = () => (
  <Card style={{marginTop: '2rem'}}
  >
    <CardHeader
      title='Disclaimer'
      actAsExpander={true}
      showExpandableButton={true}
    />
    <CardText expandable={true}>
      <div>
        <p>Este é um projeto em construção. Os valores calculados são apenas
        uma referência e não refletem valores reais.</p>
        <p>Para valores mais precisos, consulte a legislação vigente ou
        um contador.</p>
        <h5>O cálculo de CLT considera:</h5>
        <ul>
          <li>Descontos de INSS e IRRF</li>
          <li>11 salários normais, 1 salário com adicional de 1/3 de férias
          e 13º salário</li>
          <li>O resultado são valores líquidos recebidos. Para tomar decisões
            é importante considerar benefícios e outros descontos</li>
        </ul>
        <h5>O cálculo de PJ considera:</h5>
        <ul>
          <li>Pessoa jurídica do tipo MEI</li>
          <li>12 recebimentos ao ano</li>
          <li>Desconto mensal da DAS</li>
          <li>Esta forma de contratação não é estritamente legal mas é uma
          prática de mercado.</li>
        </ul>
        <h5>Features sendo desenvolvidas:</h5>
        <ul>
          <li>Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
          </li>
          <li>Adicionar à PJ custos de alimentação e transporte</li>
          <li>Adicionar opções de PJ LTDA com suas formas de tributação</li>
          <li>Exibir os cálculos, para mostrar como é difícil chegar nesses
            números</li>
        </ul>
      </div>
    </CardText>
  </Card>
);

export default Disclaimer
