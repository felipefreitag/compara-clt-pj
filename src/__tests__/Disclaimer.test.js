import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Disclaimer from '../Disclaimer'

Enzyme.configure({ adapter: new Adapter() })

it('renders correctly', () => {
  const wrapper = shallow(<Disclaimer />)
  expect(wrapper).toMatchSnapshot()
})
