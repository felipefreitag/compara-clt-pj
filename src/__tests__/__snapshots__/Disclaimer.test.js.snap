// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Disclaimer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <CardHeader
          actAsExpander={true}
          avatar={null}
          showExpandableButton={true}
          title="Disclaimer"
/>,
        <CardText
          expandable={true}
>
          <div>
                    <p>
                              Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.
                    </p>
                    <p>
                              Para valores mais precisos, consulte a legislação vigente ou um contador.
                    </p>
                    <h5>
                              O cálculo de CLT considera:
                    </h5>
                    <ul>
                              <li>
                                        Descontos de INSS e IRRF
                              </li>
                              <li>
                                        11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário
                              </li>
                              <li>
                                        O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos
                              </li>
                    </ul>
                    <h5>
                              O cálculo de PJ considera:
                    </h5>
                    <ul>
                              <li>
                                        Pessoa jurídica do tipo MEI
                              </li>
                              <li>
                                        12 recebimentos ao ano
                              </li>
                              <li>
                                        Desconto mensal da DAS
                              </li>
                              <li>
                                        Esta forma de contratação não é estritamente legal mas é uma prática de mercado.
                              </li>
                    </ul>
                    <h5>
                              Features sendo desenvolvidas:
                    </h5>
                    <ul>
                              <li>
                                        Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
                              </li>
                              <li>
                                        Adicionar à PJ custos de alimentação e transporte
                              </li>
                              <li>
                                        Adicionar opções de PJ LTDA com suas formas de tributação
                              </li>
                              <li>
                                        Exibir os cálculos, para mostrar como é difícil chegar nesses números
                              </li>
                    </ul>
          </div>
</CardText>,
      ],
      "expandable": false,
      "expanded": null,
      "initiallyExpanded": false,
      "style": Object {
        "marginTop": "2rem",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "actAsExpander": true,
          "avatar": null,
          "showExpandableButton": true,
          "title": "Disclaimer",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "children": <div>
            <p>
                        Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.
            </p>
            <p>
                        Para valores mais precisos, consulte a legislação vigente ou um contador.
            </p>
            <h5>
                        O cálculo de CLT considera:
            </h5>
            <ul>
                        <li>
                                    Descontos de INSS e IRRF
                        </li>
                        <li>
                                    11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário
                        </li>
                        <li>
                                    O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos
                        </li>
            </ul>
            <h5>
                        O cálculo de PJ considera:
            </h5>
            <ul>
                        <li>
                                    Pessoa jurídica do tipo MEI
                        </li>
                        <li>
                                    12 recebimentos ao ano
                        </li>
                        <li>
                                    Desconto mensal da DAS
                        </li>
                        <li>
                                    Esta forma de contratação não é estritamente legal mas é uma prática de mercado.
                        </li>
            </ul>
            <h5>
                        Features sendo desenvolvidas:
            </h5>
            <ul>
                        <li>
                                    Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
                        </li>
                        <li>
                                    Adicionar à PJ custos de alimentação e transporte
                        </li>
                        <li>
                                    Adicionar opções de PJ LTDA com suas formas de tributação
                        </li>
                        <li>
                                    Exibir os cálculos, para mostrar como é difícil chegar nesses números
                        </li>
            </ul>
</div>,
          "expandable": true,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.
</p>,
              <p>
                Para valores mais precisos, consulte a legislação vigente ou um contador.
</p>,
              <h5>
                O cálculo de CLT considera:
</h5>,
              <ul>
                <li>
                                Descontos de INSS e IRRF
                </li>
                <li>
                                11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário
                </li>
                <li>
                                O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos
                </li>
</ul>,
              <h5>
                O cálculo de PJ considera:
</h5>,
              <ul>
                <li>
                                Pessoa jurídica do tipo MEI
                </li>
                <li>
                                12 recebimentos ao ano
                </li>
                <li>
                                Desconto mensal da DAS
                </li>
                <li>
                                Esta forma de contratação não é estritamente legal mas é uma prática de mercado.
                </li>
</ul>,
              <h5>
                Features sendo desenvolvidas:
</h5>,
              <ul>
                <li>
                                Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
                </li>
                <li>
                                Adicionar à PJ custos de alimentação e transporte
                </li>
                <li>
                                Adicionar opções de PJ LTDA com suas formas de tributação
                </li>
                <li>
                                Exibir os cálculos, para mostrar como é difícil chegar nesses números
                </li>
</ul>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.",
              },
              "ref": null,
              "rendered": "Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Para valores mais precisos, consulte a legislação vigente ou um contador.",
              },
              "ref": null,
              "rendered": "Para valores mais precisos, consulte a legislação vigente ou um contador.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "O cálculo de CLT considera:",
              },
              "ref": null,
              "rendered": "O cálculo de CLT considera:",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    Descontos de INSS e IRRF
</li>,
                  <li>
                    11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário
</li>,
                  <li>
                    O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos
</li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Descontos de INSS e IRRF",
                  },
                  "ref": null,
                  "rendered": "Descontos de INSS e IRRF",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário",
                  },
                  "ref": null,
                  "rendered": "11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos",
                  },
                  "ref": null,
                  "rendered": "O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "O cálculo de PJ considera:",
              },
              "ref": null,
              "rendered": "O cálculo de PJ considera:",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    Pessoa jurídica do tipo MEI
</li>,
                  <li>
                    12 recebimentos ao ano
</li>,
                  <li>
                    Desconto mensal da DAS
</li>,
                  <li>
                    Esta forma de contratação não é estritamente legal mas é uma prática de mercado.
</li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Pessoa jurídica do tipo MEI",
                  },
                  "ref": null,
                  "rendered": "Pessoa jurídica do tipo MEI",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "12 recebimentos ao ano",
                  },
                  "ref": null,
                  "rendered": "12 recebimentos ao ano",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Desconto mensal da DAS",
                  },
                  "ref": null,
                  "rendered": "Desconto mensal da DAS",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Esta forma de contratação não é estritamente legal mas é uma prática de mercado.",
                  },
                  "ref": null,
                  "rendered": "Esta forma de contratação não é estritamente legal mas é uma prática de mercado.",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Features sendo desenvolvidas:",
              },
              "ref": null,
              "rendered": "Features sendo desenvolvidas:",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
</li>,
                  <li>
                    Adicionar à PJ custos de alimentação e transporte
</li>,
                  <li>
                    Adicionar opções de PJ LTDA com suas formas de tributação
</li>,
                  <li>
                    Exibir os cálculos, para mostrar como é difícil chegar nesses números
</li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR",
                  },
                  "ref": null,
                  "rendered": "Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Adicionar à PJ custos de alimentação e transporte",
                  },
                  "ref": null,
                  "rendered": "Adicionar à PJ custos de alimentação e transporte",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Adicionar opções de PJ LTDA com suas formas de tributação",
                  },
                  "ref": null,
                  "rendered": "Adicionar opções de PJ LTDA com suas formas de tributação",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Exibir os cálculos, para mostrar como é difícil chegar nesses números",
                  },
                  "ref": null,
                  "rendered": "Exibir os cálculos, para mostrar como é difícil chegar nesses números",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
          ],
          "type": "div",
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <CardHeader
            actAsExpander={true}
            avatar={null}
            showExpandableButton={true}
            title="Disclaimer"
/>,
          <CardText
            expandable={true}
>
            <div>
                        <p>
                                    Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.
                        </p>
                        <p>
                                    Para valores mais precisos, consulte a legislação vigente ou um contador.
                        </p>
                        <h5>
                                    O cálculo de CLT considera:
                        </h5>
                        <ul>
                                    <li>
                                                Descontos de INSS e IRRF
                                    </li>
                                    <li>
                                                11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário
                                    </li>
                                    <li>
                                                O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos
                                    </li>
                        </ul>
                        <h5>
                                    O cálculo de PJ considera:
                        </h5>
                        <ul>
                                    <li>
                                                Pessoa jurídica do tipo MEI
                                    </li>
                                    <li>
                                                12 recebimentos ao ano
                                    </li>
                                    <li>
                                                Desconto mensal da DAS
                                    </li>
                                    <li>
                                                Esta forma de contratação não é estritamente legal mas é uma prática de mercado.
                                    </li>
                        </ul>
                        <h5>
                                    Features sendo desenvolvidas:
                        </h5>
                        <ul>
                                    <li>
                                                Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
                                    </li>
                                    <li>
                                                Adicionar à PJ custos de alimentação e transporte
                                    </li>
                                    <li>
                                                Adicionar opções de PJ LTDA com suas formas de tributação
                                    </li>
                                    <li>
                                                Exibir os cálculos, para mostrar como é difícil chegar nesses números
                                    </li>
                        </ul>
            </div>
</CardText>,
        ],
        "expandable": false,
        "expanded": null,
        "initiallyExpanded": false,
        "style": Object {
          "marginTop": "2rem",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "actAsExpander": true,
            "avatar": null,
            "showExpandableButton": true,
            "title": "Disclaimer",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "children": <div>
              <p>
                            Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.
              </p>
              <p>
                            Para valores mais precisos, consulte a legislação vigente ou um contador.
              </p>
              <h5>
                            O cálculo de CLT considera:
              </h5>
              <ul>
                            <li>
                                          Descontos de INSS e IRRF
                            </li>
                            <li>
                                          11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário
                            </li>
                            <li>
                                          O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos
                            </li>
              </ul>
              <h5>
                            O cálculo de PJ considera:
              </h5>
              <ul>
                            <li>
                                          Pessoa jurídica do tipo MEI
                            </li>
                            <li>
                                          12 recebimentos ao ano
                            </li>
                            <li>
                                          Desconto mensal da DAS
                            </li>
                            <li>
                                          Esta forma de contratação não é estritamente legal mas é uma prática de mercado.
                            </li>
              </ul>
              <h5>
                            Features sendo desenvolvidas:
              </h5>
              <ul>
                            <li>
                                          Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
                            </li>
                            <li>
                                          Adicionar à PJ custos de alimentação e transporte
                            </li>
                            <li>
                                          Adicionar opções de PJ LTDA com suas formas de tributação
                            </li>
                            <li>
                                          Exibir os cálculos, para mostrar como é difícil chegar nesses números
                            </li>
              </ul>
</div>,
            "expandable": true,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <p>
                  Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.
</p>,
                <p>
                  Para valores mais precisos, consulte a legislação vigente ou um contador.
</p>,
                <h5>
                  O cálculo de CLT considera:
</h5>,
                <ul>
                  <li>
                                    Descontos de INSS e IRRF
                  </li>
                  <li>
                                    11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário
                  </li>
                  <li>
                                    O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos
                  </li>
</ul>,
                <h5>
                  O cálculo de PJ considera:
</h5>,
                <ul>
                  <li>
                                    Pessoa jurídica do tipo MEI
                  </li>
                  <li>
                                    12 recebimentos ao ano
                  </li>
                  <li>
                                    Desconto mensal da DAS
                  </li>
                  <li>
                                    Esta forma de contratação não é estritamente legal mas é uma prática de mercado.
                  </li>
</ul>,
                <h5>
                  Features sendo desenvolvidas:
</h5>,
                <ul>
                  <li>
                                    Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
                  </li>
                  <li>
                                    Adicionar à PJ custos de alimentação e transporte
                  </li>
                  <li>
                                    Adicionar opções de PJ LTDA com suas formas de tributação
                  </li>
                  <li>
                                    Exibir os cálculos, para mostrar como é difícil chegar nesses números
                  </li>
</ul>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.",
                },
                "ref": null,
                "rendered": "Este é um projeto em construção. Os valores calculados são apenas uma referência e não refletem valores reais.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Para valores mais precisos, consulte a legislação vigente ou um contador.",
                },
                "ref": null,
                "rendered": "Para valores mais precisos, consulte a legislação vigente ou um contador.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "O cálculo de CLT considera:",
                },
                "ref": null,
                "rendered": "O cálculo de CLT considera:",
                "type": "h5",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li>
                      Descontos de INSS e IRRF
</li>,
                    <li>
                      11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário
</li>,
                    <li>
                      O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos
</li>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Descontos de INSS e IRRF",
                    },
                    "ref": null,
                    "rendered": "Descontos de INSS e IRRF",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário",
                    },
                    "ref": null,
                    "rendered": "11 salários normais, 1 salário com adicional de 1/3 de férias e 13º salário",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos",
                    },
                    "ref": null,
                    "rendered": "O resultado são valores líquidos recebidos. Para tomar decisões é importante considerar benefícios e outros descontos",
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "O cálculo de PJ considera:",
                },
                "ref": null,
                "rendered": "O cálculo de PJ considera:",
                "type": "h5",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li>
                      Pessoa jurídica do tipo MEI
</li>,
                    <li>
                      12 recebimentos ao ano
</li>,
                    <li>
                      Desconto mensal da DAS
</li>,
                    <li>
                      Esta forma de contratação não é estritamente legal mas é uma prática de mercado.
</li>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Pessoa jurídica do tipo MEI",
                    },
                    "ref": null,
                    "rendered": "Pessoa jurídica do tipo MEI",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "12 recebimentos ao ano",
                    },
                    "ref": null,
                    "rendered": "12 recebimentos ao ano",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Desconto mensal da DAS",
                    },
                    "ref": null,
                    "rendered": "Desconto mensal da DAS",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Esta forma de contratação não é estritamente legal mas é uma prática de mercado.",
                    },
                    "ref": null,
                    "rendered": "Esta forma de contratação não é estritamente legal mas é uma prática de mercado.",
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Features sendo desenvolvidas:",
                },
                "ref": null,
                "rendered": "Features sendo desenvolvidas:",
                "type": "h5",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li>
                      Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR
</li>,
                    <li>
                      Adicionar à PJ custos de alimentação e transporte
</li>,
                    <li>
                      Adicionar opções de PJ LTDA com suas formas de tributação
</li>,
                    <li>
                      Exibir os cálculos, para mostrar como é difícil chegar nesses números
</li>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR",
                    },
                    "ref": null,
                    "rendered": "Adicionar outros benefícios e descontos à CLT, como FGTS, VT, VR",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Adicionar à PJ custos de alimentação e transporte",
                    },
                    "ref": null,
                    "rendered": "Adicionar à PJ custos de alimentação e transporte",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Adicionar opções de PJ LTDA com suas formas de tributação",
                    },
                    "ref": null,
                    "rendered": "Adicionar opções de PJ LTDA com suas formas de tributação",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Exibir os cálculos, para mostrar como é difícil chegar nesses números",
                    },
                    "ref": null,
                    "rendered": "Exibir os cálculos, para mostrar como é difícil chegar nesses números",
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
            ],
            "type": "div",
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
